test/install-binary-dependencies.sh

```diff
@@ -4,7 +4,7 @@ clib install -c stephenmathieson/tabs-to-spaces@1.0.0 > /dev/null || {
   echo >&2 "Failed to install stephenmathieson/tabs-to-spaces"
   exit 1
 }
-command -v t2s >/dev/null 2>&1 || {
+command -v $PREFIX/bin/t2s >/dev/null 2>&1 || {
```

Comment:
This breaks the test for me on OSX (and also linux?).  What was it supposed to fix?

Reply:
looks like a test for installing a binary command and its deps, what is breaking on your end?

Reply:
I get `Failed to put t2s on path` when using the `$PREFIX/bin/` version.
`command -v t2s >/dev/null 2>&1` version passes.  
Does a $PREFIX env var need to be set somewhere?

Reply:
probably. I'll take a look!